src/commands/notifications.ts

```diff
@@ -0,0 +1,113 @@
+import ora from 'ora';
+import addDependency from '../util/addDependency';
+import copyTemplateDirectory from '../util/copyTemplateDirectory';
+import exec from '../util/exec';
+import isExpo from '../util/isExpo';
+import addExpoConfig from '../util/addExpoConfig';
+import injectHooks from '../util/injectHooks';
+import readAppJson from '../util/readAppJson';
+import { input } from '@inquirer/prompts';
+import commit from '../util/commit';
+
+const handleCommitError = (error: { stdout: string }) => {
```

Comment:
The goal of this handler was specific to the case where the `notifications` command is run two times. There would be `nothing to commit` and the command would fail midway without trying the next steps that may or may not cause changes (depending on any user-made changes).

This happened more often when I was testing the command repeatedly during development but I figured it would be good to handle the edge case.

I removed the ESLint override, which was unnecessary.

The intention there is for it to just handle the `nothing to commit` case which has a `stdout`, every other type of error should throw as usual.